webapp/src/components/task_creation_form/TaskCreationForm.vue

```diff
@@ -364,12 +364,38 @@
             >
               <div v-show="differentialPrivacy" class="flex flex-col">
                 <FormLabel
-                  label="Standard deviation of the noise"
+                  label="Privacy budget (ε): Controls how much noise is added for differential privacy, smaller ε means stronger privacy but more noise added to the weight update"
                   type="required"
                 >
                   <FormField
                     name="trainingInformation.privacy.noiseScale"
-                    placeholder="2"
+                    placeholder="1"
+                    as="input"
+                    type="number"
+                  />
+                </FormLabel>
+              </div>
+              <div v-show="differentialPrivacy" class="flex flex-col">
+                <FormLabel
+                  label="Error parameter (δ): Small probability that the privacy guarantee may not hold"
+                  type="required"
+                >
+                  <FormField
+                    name="trainingInformation.privacy.delta"
+                    placeholder="1e-5"
+                    as="input"
+                    type="number"
+                  />
+                </FormLabel>
+              </div>
+              <div v-show="differentialPrivacy" class="flex flex-col">
```

Comment:
you can keep the same div block for the whole option, it's simpler and ~links the elements tgth

```suggestion

                <FormLabel
                  label="Error parameter (δ): Small probability that the privacy guarantee may not hold"
                  type="required"
                >
                  <FormField
                    name="trainingInformation.privacy.delta"
                    placeholder="1e-5"
                    as="input"
                    type="number"
                  />
                </FormLabel>

```